tests/tf-manifests/rhcs/cluster-autoscaler/main.tf

```diff
@@ -0,0 +1,61 @@
+terraform {
+  required_providers {
+    rhcs = {
+      version = ">= 1.1.0"
+      source  = "terraform.local/local/rhcs"
+    }
+  }
+}
+
+provider "rhcs" {
+  url   = var.url
+}
+
+resource "rhcs_cluster_autoscaler" "cluster_autoscaler" {
+  cluster                       = var.cluster_id
+  balance_similar_node_groups   = true
+  skip_nodes_with_local_storage = true
```

Comment:
@sunzhaohua2 You are hardcode all values without set from variables, then it's impossible to test via service for negative scenarios. Please update to like `skip_nodes_with_local_storage =var.skip_nodes_with_local_storage`